src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue

```diff
@@ -169,6 +169,17 @@
 					</template>
 					{{ t('spreed', 'Grant all permissions') }}
 				</ActionButton>
+				<ActionButton
```

Comment:
I would put it bottom (or top), not make a conditional button appear in the middle of the list and you always will have to search

Reply:
It's not conditional atm, but ok will put it on top

Reply:
Actually this will be a radio soon and it probably makes sense if it stays in the middle, because on the permissions scale it is in between

Reply:
So, I would not show this, if the attendee does not have custom permissions set for themselves but is currently following "call/conversation" permissions. And that's why I would then call it "reset" and have it at the end of the list.
It might or might not be bigger/smaller than Grant all/none, bout could also be very much one of that. But it's more about the fact that the user will now follow global rules again and their custom permission icon will vanish.

Reply:
So what do we do? as long as it's not a radio I guess it makes more sense to have the condition on the button, show it at the end of the list and use the wording "reset".
If it would be a radio default in the middle might make more sense, but we are running out of time.

Therefor my proposal would be the button approach for now. Have the commit available locally already:
Default | Non-default
---|---
![Bildschirmfoto von 2021-11-10 09-35-29](https://user-images.githubusercontent.com/213943/141078443-88769092-06a8-4e66-a57b-9520700a3297.png) | ![Bildschirmfoto von 2021-11-10 09-32-01](https://user-images.githubusercontent.com/213943/141078448-973d5813-9e3f-4dec-b52c-5f07abd81efe.png)